controllers/compassmanager_controller.go

```diff
@@ -251,19 +249,12 @@ func (cm *CompassManagerReconciler) handleKymaDeletion(name types.NamespacedName
 	return nil
 }
 
-func (cm *CompassManagerReconciler) makeNewCompassMappingAndRequeue(kymaName types.NamespacedName, kymaAnnotations map[string]string) (ctrl.Result, error) {
+func (cm *CompassManagerReconciler) makeNewCompassMappingAndRequeue(kymaName types.NamespacedName) (ctrl.Result, error) {
 	var runtimeID string
 
 	// default mode - application-connector module is enabled for the first time in Kyma, we create Compass Manager Mapping
 	runtimeRegistrationType := "newly provisioned Kyma runtime"
 
-	// remove below after migration is completed
-	if migrationCompassRuntimeID, ok := kymaAnnotations[AnnotationIDForMigration]; ok && len(migrationCompassRuntimeID) > 0 {
-		// Runtime registered previously by Provisioner, but we have the Compass ID provided by KEB
-		runtimeRegistrationType = "already registered Kyma runtime"
-		runtimeID = migrationCompassRuntimeID
-	}
-
 	cm.Log.Infof("Attempting to create Compass Manager Mapping for %s for Kyma resource %s.", runtimeRegistrationType, kymaName.Name)
 	cmerr := cm.cluster.CreateCompassMapping(kymaName, runtimeID)
```

Comment:
this is the only place where we call `CreateCompassMapping`, and the `runtimeID` is always an empty string. Could we remove that argument?

Reply:
Done!